# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/player/tests/data/empty.wgsl
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/player/tests/data/empty.wgsl b/gfx/wgpu/player/tests/data/empty.wgsl
deleted file mode 100644
--- a/gfx/wgpu/player/tests/data/empty.wgsl
+++ /dev/null
@@ -1,3 +0,0 @@
-[[stage(compute), workgroup_size(1)]]
-fn main() {
-}